lib/coinbase/address.rb

```diff
@@ -162,6 +162,29 @@ def export
       @key.private_hex
     end
 
+    # Lists the IDs of all Transfers associated with the given Wallet and Address.
+    # @return [Array<String>] The IDs of all Transfers belonging to the Wallet and Address
+    def list_transfer_ids
+      transfer_ids = []
+      page = nil
+
+      loop do
+        opts = { limit: 100, page: page }
```

Comment:
Do we need a separate variable for this?

Reply:
I can update to inline if you prefer

Reply:
updated in most recent commit